wire-library/wire-schema/src/jvmTest/kotlin/com/squareup/wire/schema/internal/parser/ProtoParserTest.kt

```diff
@@ -2413,13 +2397,22 @@ class ProtoParserTest {
     assertThat(ProtoParser.parse(location, proto)).isEqualTo(expected)
   }
 
-  @Test @Ignore("Broken. See https://github.com/square/wire/issues/1797")
-  fun protoKeywordAsEnumConstants() {
+  @Test fun protoKeywordAsEnumConstants() {
+    // Note: cases were protoc fails are commented out.
```

Comment:
lets make Wire consistent with protoc here? Totally sucks to define an enum, use it with Wire, and then to find out your schema isn’t portable to protoc

Reply:
I see. Need to make sure then fail in both or pass in both.

Reply:
And yes they are both behaving the same.